### PR TITLE
Java Assignment-2

### PR DESCRIPTION
**Pull request by 890534**
My week2 Assignment-Sorting, Abstract, Interface , Pair and sum

### CODE REVIEW

File: Week-2/InterfaceExample/InterfaceExample.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+public class InterfaceExample {
+
+    public static void main(String[] args) {
+
+       Bike bike= new RoyalEnfield();
+        bike.Company();
+        bike.SpeedLimit();
+        bike = new HarleyDavidson();
+        bike.Company();
```
Comment (VBhaskarachary) [inline Week-2/InterfaceExample/InterfaceExample.java]: Please give input and compare with  restrict the speed limit of a bike to 90 KMPH and display proper message

File: Week-2/SortAndSearch/SortAndSearch.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import java.util.*;
+
+public class SortAndSearch {
+    static List<Number> numbers = new ArrayList<>();
+    static Set<Number> noDuplicate = new HashSet<>();
+
+    public List<Number> removeDuplicates(List<Number> numbers) {
+        List<Number> originalList = new ArrayList<>();
+        noDuplicate.addAll(numbers);
+        originalList.addAll(noDuplicate);
```
Comment (VBhaskarachary) [inline Week-2/SortAndSearch/SortAndSearch.java]: Use proper return type for removing unnecessary code

File: Week-2/InterfaceExample/InterfaceExample.java (hunk 1 above)
Comment (890534) [reply Week-2/InterfaceExample/InterfaceExample.java]: code is fixed

File: Week-2/SortAndSearch/SortAndSearch.java (hunk 2 above)
Comment (890534) [reply Week-2/SortAndSearch/SortAndSearch.java]: code is fixed

**Review by VBhaskarachary — APPROVED** [review]
Changes looks good

**Pull request merged**